python/lsst/rubintv/production/rubinTv.py

```diff
@@ -1402,46 +1411,55 @@ def runM1M3HardpointAnalysis(self, event):
                                           isLiveFile=False
                                           )
 
-    def processDay(self, dayObs, skipEventNumbers={}):
+    def processDay(self, dayObs):
         """
         """
         events = self.eventMaker.getEvents(dayObs)
-        self.log.info(f'Found {len(events)} events for {dayObs=} of which'
-                      f' {len(skipEventNumbers)} have already been created')
-        mountMotionPlotted = set()
-        icsPlotted = set()
         for event in events:
-            if event.seqNum in skipEventNumbers:
-                continue
             assert event.dayObs == dayObs
+
+            nMountMotionPlots = len(self.plotsMade['MountMotionAnalysis'])
+            nM1M3HardpointPlots = len(self.plotsMade['M1M3HardpointAnalysis'])
+            # the interesting phrasing in the message is because these plots
+            # don't necessarily exist, due either to failures or M1M3 analyses
+            # only being valid for some events so this is to make it clear
+            # they've been processed.
+            self.log.info(f'Found {len(events)} events for {dayObs=} of which '
```

Comment:
So `getEvents` returns all events for the day, and actually does the calculation/generation, so that `len(events)` will be all the events for the current day, and so is strictly ≥ either of those other numbers.